File: Balance_Manager/src/main/java/com/greenland/balancemanager/domain/TxDataRow.java
package com.greenland.balancemanager.domain;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.UUID;

import lombok.Builder;
import lombok.Getter;


@Builder
@Getter
public class TxDataRow {

	private UUID uuid;
	private LocalDate txDate;
	private String accountName;
	private String categoryName;
	private BigDecimal debitAmount;
	private BigDecimal creditAmount;
	private boolean isReconsiled;
	private boolean isRemote;
	private LocalDate created_At;
	private LocalDate updated_At;
	
}

File: balance_manager_react/src/components/TxDataRow/TxDataRow.js
import React from "react";
import classes from "./TxDataRow.module.css";

const txDataRow = (props) => (
  <article className={classes.TxDataRow}>
    <h2>{props.txDataRow.txDate}</h2>
    <hr />
    <h3>{props.txDataRow.accountName}</h3>
    <h3>{props.txDataRow.categoryName}</h3>
    <h3>{props.txDataRow.debitAmount}</h3>
    <h3>{props.txDataRow.creditAmount}</h3>
    <h3 className={classes.Author}>{props.txDataRow.isRemote}</h3>
  </article>
);

export default txDataRow;
